src/components/Company/Offers/Manage/CompanyOffersManagementWidget.spec.js

```diff
@@ -0,0 +1,33 @@
+import React from "react";
+import { screen } from "@testing-library/react";
+
+import CompanyOffersManagementWidget from "./CompanyOffersManagementWidget";
+import { renderWithStore } from "../../../../test-utils";
+import { MuiPickersUtilsProvider } from "@material-ui/pickers";
+import DateFnsUtils from "@date-io/date-fns";
+
+
+describe("App", () => {
+    test("Renders Component", async () => {
+        renderWithStore(
+            <MuiPickersUtilsProvider utils={DateFnsUtils}>
+                <CompanyOffersManagementWidget />
+            </MuiPickersUtilsProvider>
+        );
+
+        // Use getBy by default when element should be available
+        expect(screen.getByText("Loading...")).toBeInTheDocument();
+
+        expect(screen.getByRole("heading")).toBeInTheDocument();
```

Comment:
Avoid using getByRole queries as they are too slow. besides this query is too generic, I believe.

If the heading you are referring to here is the same that shows "Loading...", you can remove this line, since you are testing that above

Reply:
Yes, I just left it there as I was testing react-native-testing-library functionalities. I will try to make more thorough tests